src/test/java/core/basesyntax/strategy/OperationHandlerTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.handlers.BalanceHandlerImpl;
+import core.basesyntax.handlers.PurchaseHandlerImpl;
+import core.basesyntax.handlers.ReturnHandlerImpl;
+import core.basesyntax.handlers.SupplyHandlerImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationHandlerTest {
+    private static OperationHandler operationHandler;
```

Comment:
```suggestion
    private static OperationHandler returnOperationHandler;
    private static OperationHandler supplyOperationHandler;
    private static OperationHandler purchaseOperationHandler;
    private static OperationHandler balanceOperationHandler;
```
and move initialization to @BeforeClass method